docs/pages/build-reference/variants.mdx

```diff
@@ -79,13 +79,31 @@ export default {
   },
   android: {
     package: IS_DEV ? 'com.myapp.dev' : 'com.myapp',
-  },
+  }
 };
 ```
 
 In the above example, the environment variable `IS_DEV` is used to differentiate between the development and production environment. Based on its value, the different Application IDs or Bundle Identifiers are set for each variant.
 
-> **Note**: If you are using any libraries that require you to register your application identifier with an external service to use the SDK, such as Google Maps or Firebase Cloud Messaging (FCM), you'll need to have a separate configuration for that API for the `android.package` and `ios.bundleIdentifier`. You can also swap this configuration using the same approach as above.
+#### Additional app variant customizations
+
+You may wish to customize other aspects of your app on a per-variant basis. You can swap any configuration that you used previously in **app.json** using the same approach as above.
+
+Some examples:
+
+- If you are using any libraries that require you to register your application identifier with an external service to use the SDK, such as Google Maps or Firebase Cloud Messaging (FCM), you'll need to have a separate configuration for that API for the `android.package` and `ios.bundleIdentifier`.
+- If you are using [development builds](/develop/development-builds/introduction/) you can set the `expo-dev-client` config plugin to disable the app scheme used by Expo CLI and EAS Update QR codes on your non-development build, so those URL's will always launch the development build regardless of your device defaults:
+
+```js app.config.js
+plugins: [
+  [
+    'expo-dev-client',
+    {
+      disableDevClientScheme: !IS_DEV,
```

Comment:
```suggestion
      addGeneratedScheme: IS_DEV,
```

maybe something like this? i think it's nice to use the positive form of this and have people opt out of it, since we tend to use "enabled" rather than "disabled" in our properties - "disabled: !THING` is slightly harder to reason about than `enabled: THING`